src/Fantomas.Core/CodePrinter.fs

```diff
@@ -96,7 +98,7 @@ let genTrivia (trivia: TriviaNode) (ctx: Context) =
             +> ifElse after sepNlnForTrivia sepNone
         | CommentOnSingleLine s
         | Directive s -> (ifElse addNewline sepNlnForTrivia sepNone) +> !-s +> sepNlnForTrivia
-        | Newline -> (ifElse addNewline (sepNlnForTrivia +> sepNlnForTrivia) sepNlnForTrivia)
+        | Newline -> ifElse addNewline (sepNlnForTrivia +> sepNlnForTrivia) sepNlnForTrivia
```

Comment:
@majocha nit, this hunk seems to be not needed?

Reply:
yep, remnants of experimenting 🙃